spec/zoneddatetime.html

```diff
@@ -1303,8 +1316,11 @@ <h1>
           As specified in <a href="https://tools.ietf.org/html/rfc5545">RFC 5545</a>, the date portion of the duration is added in calendar days, and the time portion is added in exact time.
         </dd>
       </dl>
-      <p>Unless _precalculatedPlainDateTime_ is supplied, the given _timeZone_'s `getOffsetNanosecondsFor` method will be called to convert _epochNanoseconds_ to a wall-clock time.</p>
+      <p>Unless _precalculatedPlainDateTime_ is supplied, the given _timeZone_'s `getOffsetNanosecondsFor` method will be called to convert _epochNanoseconds_ to a wall-clock time, and _timeZoneRec_ must have looked up `getOffsetNanosecondsFor`.</p>
       <emu-alg>
+        1. Assert: TimeZoneMethodsRecordHasLookedUp(_timeZoneRec_, ~getPossibleInstantsFor~) is *true*.
```

Comment:
The descriptive paragraphs aren't trying to completely cover the assertions, but I will add an extra sentence here since it's odd that there's a prose sentence about one method but not the other.